bucket/impregnate.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.2.2",
+    "description": "A standalone installer for Cumcord.",
+    "homepage": "https://github.com/Cumcord/Impregnate",
+    "license": "Apache-2.0",
+    "url": "https://github.com/Cumcord/Impregnate/releases/download/v1.2.2/impregnate.exe",
+    "hash": "AE00DB169C649428DA2BC7BAEA1ACAAA0290C95FB1DE9852F3CB9C63A5E4B337",
+    "bin": "impregnate.exe",
+    "shortcuts": [
+        [
+            "impregnate.exe",
+            "Impregnate"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/Cumcord/Impregnate/releases/latest",
+        "regex": "v(\\d\\.\\d+\\.\\d+),"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Cumcord/Impregnate/releases/download/$version/impregnate.exe"
```

Comment:
Could you check if this application works with checkver.ps1 offline once with ForceUpdate parameter

https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#using-checkverps1-to-query-and-autoupdate